imagebuildah/build.go

```diff
@@ -1348,12 +1351,20 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (strin
 			b.imageMap[stage.Name] = imgID
 		}
 	}
+	if len(b.unusedArgs) > 0 {
+		unusedList := make([]string, 0, len(b.unusedArgs))
+		for k := range b.unusedArgs {
+			unusedList = append(unusedList, k)
+		}
+		fmt.Fprintf(b.out, "[Warning] one or more build args %v were not consumed\n", unusedList)
```

Comment:
Nit of a nit: can we move printing the list to the end of the sentence?

"[Warning] one or more build args were not consumed: %v\n"

I think this is a bit easier to read for longer lists.

Reply:
Okay, the list should be at the end of the line now.  We populate it using the keys from the map, so their order wasn't well-defined, so we sort it now, too.